python/cugraph/cugraph/gnn/graph_store.py

```diff
@@ -54,9 +54,9 @@ def add_node_data(
         self,
         df,
         node_col_name,
-        feat_name=None,
         ntype=None,
-        is_single_vector_feature=True,
+        feat_name=None,
+        contains_vector_features=False,
     ):
```

Comment:
Changed to this to keep the API cleaner and intuitive as you would want to provide feat_name together with contains_vector_features